sycl-jit/jit-compiler/lib/rtc/DeviceCompilation.h

```diff
@@ -17,15 +17,18 @@
 #include <llvm/Support/Error.h>
 
 #include <memory>
+#include <string>
 
 namespace jit_compiler {
 
 llvm::Expected<std::unique_ptr<llvm::Module>>
 compileDeviceCode(InMemoryFile SourceFile, View<InMemoryFile> IncludeFiles,
-                  const llvm::opt::InputArgList &UserArgList);
+                  const llvm::opt::InputArgList &UserArgList,
+                  std::string &BuildLog);
```

Comment:
Can we use `std::string` in this header for ABI purposes?

Reply:
This header is internal to the JIT library, so no ABI concerns here.